src/components/ProjectSearch.js

```diff
@@ -84,13 +82,18 @@ function sanitizeQuery(query: string): string {
   return query.replace(/\\$/, "");
 }
 
+type FileItem = {
+  setExpanded: (Result, boolean) => any,
```

Comment:
Better to go with `mixed` instead of `any` for things like this so attempting to use the return value would require refinement.